credits.md

```diff
@@ -1,5 +1,7 @@
 ### Credits
-
+- [CI greetings Bot setup](https://github.com/Ansub/SyntaxUI/pull/182) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
+- [Issue btns and corresponding forms](https://github.com/Ansub/SyntaxUI/pull/72) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
+- [Automated Issue labeling](https://github.com/Ansub/SyntaxUI/pull/78) - [Daksh Singh Rathore](https://twitter.com/dakshsinghrath9)
```

Comment:
add your credits at the bottom and also use short names for names of the features and please don't use abbreviation